### PR TITLE
add info-component

### PR DESCRIPTION
**Pull request by Ponomaleks**

### CODE REVIEW

File: src/app/main/pages/info/info.component.html [hunk 1]
```diff
@@ -1 +1,72 @@
...
+        class="datepicker__wrapper"
+        data-placeholder="Date of birth"
+        [ngClass]="{ filled: !!datepickerInput.value }"
+      >
+        <input #datepickerInput class="datepicker" autocomplete="off" [matDatepicker]="picker" />
+        <i class="datepicker__icon" (click)="picker.open()">
+          <img src="/assets/icons/calendar.svg" height="24" width="24" alt="open datepicker" (click)="picker.open()" />
+        </i>
+        <mat-datepicker #picker startView="multi-year" [startAt]="datePickerStartDate" [disabled]="false">
+        </mat-datepicker>
```
Comment (GennadiyShpak) [inline src/app/main/pages/info/info.component.html:21]:
```suggestion
        <mat-datepicker #picker startView="multi-year" [startAt]="datePickerStartDate" [disabled]="false"/>
```

File: src/app/main/service/main-api.service.ts [hunk 2]
```diff
@@ -43,8 +43,12 @@ export class MainApiService {
     );
   }
 
-  getPersonalData(): Observable<ExpandedUserDetailed> {
-    return of(USERS_MOCK.find(user => user.userId === '100')!).pipe(
+  getPersonalData(): Observable<UserDetailed> {
+    return of(USERS_MOCK.find(user => user.userId === '100')!).pipe(take(1));
+  }
```
Comment (GennadiyShpak) [inline src/app/main/service/main-api.service.ts:48]: nedd TODO comment

File: src/app/shared/components/epm-chips/epm-chips.component.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
   </div>
   <p class="description" [ngClass]="{ left: !iconName, right: !isEditable }">
     {{ chips }}
   </p>
-  <button *ngIf="isEditable" class="button" type="button" (click)="onIsEditableClick(chips)">
+  <i *ngIf="isEditable" class="button">
     <img [ngSrc]="iconFileName" width="32" height="32" alt="action icon" />
-  </button>
-</div>
+  </i>
```
Comment (GennadiyShpak) [inline src/app/shared/components/epm-chips/epm-chips.component.html:10]: why this tag?

File: src/app/main/pages/info/info.component.html (hunk 1 above)
Comment (Ponomaleks) [reply src/app/main/pages/info/info.component.html]: fixed

File: src/app/main/service/main-api.service.ts (hunk 2 above)
Comment (Ponomaleks) [reply src/app/main/service/main-api.service.ts:49]: added TODO

File: src/app/shared/components/epm-chips/epm-chips.component.html (hunk 3 above)
Comment (Ponomaleks) [reply src/app/shared/components/epm-chips/epm-chips.component.html:10]: changed to button

**Review by GennadiyShpak — APPROVED** [review]

**Pull request merged**